src/main/java/mate/academy/bookshop/model/User.java

```diff
@@ -54,7 +57,7 @@ public class User implements UserDetails {
     @JoinTable(name = "users_roles",
             joinColumns = @JoinColumn(name = "user_id"),
             inverseJoinColumns = @JoinColumn(name = "role_id"))
-    private Set<Role> roles;
+    private Set<Role> roles = new HashSet<>();
```

Comment:
Please don't include unrelated changes to PR. Better make separate PR or directrly push to main if it's something minor